extensions/wikia/Recirculation/WikiRecommendations.class.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+class WikiRecommendations {
+
+	const THUMBNAIL_WIDTH = 100;
+	const THUMBNAIL_RATIO = 16 / 9;
+
+	const RECOMMENDATIONS = [
+		'en' => [
+			[
+				'thumbnailUrl' => '/assassinscreed/images/2/25/Wikia-Visualization-Main%2Cassassinscreed.png/revision/latest/thumbnail-down/width/660/height/660?cb=20161102142202',
+				'title' => 'Assassin\'s Creed',
+				'url' => 'http://assassinscreed.wikia.com',
+			],
+			[
+				'thumbnailUrl' => '/harrypotter/images/6/64/Grindelwald-Profile.jpg/revision/latest/zoom-crop/width/1000/height/563?cb=20170425165236',
+				'title' => 'Wiki Name2',
+				'url' => 'http://gta2.wikia.com',
+			],
+			[
+				'thumbnailUrl' => '/harrypotter/images/6/64/Grindelwald-Profile.jpg/revision/latest/zoom-crop/width/1000/height/563?cb=20170425165236',
+				'title' => 'Wiki Name3',
+				'url' => 'http://gta3.wikia.com',
+			],
+		],
+		'de' => [],
+		'fr' => [],
+		'es' => [],
+		'pt-br' => [],
+		'ru' => [],
+		'it' => [],
+		'pl' => [],
+		'zh' => [],
+		'ja' => [],
+	];
+
+	const LANGUAGES = [ 'en', 'de', 'fr', 'es', 'pt-br', 'ru', 'it', 'pl', 'zh', 'ja' ];
+
+	public static function getRecommendations( $contentLanguage ) {
+		$recommendations = self::RECOMMENDATIONS['en'];
+		if ( in_array( $contentLanguage, self::LANGUAGES ) ) {
+			$recommendations = self::RECOMMENDATIONS[$contentLanguage];
+		}
+		shuffle( $recommendations );
+
+		return array_map( function ( $recommendation ) {
+			$recommendation['thumbnailUrl'] =
+				self::getThumbnailUrl( $recommendation['thumbnailUrl'] );
+
+			return $recommendation;
+		}, array_slice( $recommendations, 0, 3 ) );
+	}
+
+	private static function getThumbnailUrl( $url ) {
+		global $wgVignetteUrl;
+
+		$url = $wgVignetteUrl . $url;
+		$newUrl =
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$newUrl". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)